app/src/main/kotlin/com/wire/android/util/DateTimeUtil.kt

```diff
@@ -48,13 +48,25 @@ private val readReceiptDateTimeFormat = SimpleDateFormat(
     Locale.getDefault()
 ).apply { timeZone = TimeZone.getDefault() }
 
+private val dateTimeFormatWithDayOfWeek = SimpleDateFormat(
+    "EEE, MMM dd, hh:mm a",
```

Comment:
I just wonder how that would look in different languages.

`Day of the week, month, day of the month`

At least in Portuguese that's weird. Also AM/PM may not be followed.

We could use [DateFormat](https://developer.android.com/reference/java/text/DateFormat.html):

```kotlin
DateFormat.getDateTimeInstance(DateFormat.FULL, DateFormat.SHORT).format(date)
```

`DateFormat.FULL` for Dates will include the week day.
`DateFormat.SHORT` for Time will keep only hours and minutes.


But it should respect Locale without we needing to create different patterns for different languages.